packages/synapse-react-client/src/components/menu/ComplexMenu.tsx

```diff
@@ -22,14 +23,67 @@ export type ComplexMenuProps = {
  */
 export function ComplexMenu(props: ComplexMenuProps) {
   const { iconButtons = [], dropdownMenus = [] } = props
+  const theme = useTheme()
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'))
 
   return (
-    <Box sx={{ display: 'flex', alignItems: 'center', gap: '10px' }}>
-      {iconButtons.map(iconButton => {
-        return <IconSvgButton key={iconButton.tooltipText} {...iconButton} />
+    <Box
+      sx={theme => ({
+        display: 'flex',
+        alignItems: 'center',
+        gap: '10px',
+        [theme.breakpoints.down('sm')]: {
+          flexWrap: 'wrap',
```

Comment:
Would it make more sense to change the flex-direction to `column` here instead of making it wrap?